evalml/tests/pipeline_tests/test_graphs.py

```diff
@@ -14,14 +14,8 @@ def test_pipeline():
     class TestPipeline(BinaryClassificationPipeline):
```

Comment:
Kept this class since it was helpful to have around, but deleted `hyperparameters` because it wasn't necessary.